frontera/contrib/backends/hbase/states.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import absolute_import
+
+import logging
+
+from binascii import hexlify, unhexlify
+from collections import Iterable
+from struct import unpack
+
+from frontera.core.components import States
+from frontera.utils.misc import chunks
+
+from .utils import prepare_hbase_object
+
+
+logger = logging.getLogger(__name__)
+
+
+class HBaseState(States):
```

Comment:
This class describes states stored in hbase, so I think it should be plural: HBaseStates.